rumydata/__init__.py

```diff
@@ -35,4 +35,4 @@
 from rumydata.menu import menu
 from rumydata.table import *
 
-__version__ = '1.4.0-dev'
+__version__ = '1.5.0-dev'
```

Comment:
It sounds like this change is more of a bugfix. Let's do a patch increment rather than a minor version increment. 

1.4.1

Reply:
Good call! I've incorporated your suggested commit!